pages/campaign/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { TextField, Typography, Box, Grid, Button } from "@mui/material";
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { DateTimePicker } from '@mui/x-date-pickers';
+import { useState } from "react";
+import UploadMedia from '../../components/CreateNewCampaign/UploadMedia';
+
+const CreateNewCampaign = () => {
+    const [startDate, setStartDate] = useState(null)
+    const [endDate, setEndDate] = useState(null)
+  
```

Comment:
think u might need to add state for the other fields too. for when we press preview or save and how we will eventually send all these data to the API